index.js

```diff
@@ -176,11 +176,13 @@ module.exports = function(robot) {
 
     if (!queue.contains(user)) {
       res.reply('No sweat! You weren\'t even in the queue :)');
-    } else if (queue.isCurrent(user)) {
-      res.reply('You\'re deploying right now! Did you mean `deploy done`?');
     } else {
       queue.remove(user, areUsersEqual);
       res.reply('Alright, I took you out of the queue. Come back soon!');
+      if (!queue.isEmpty() && !queue.isCurrent(user)) {
```

Comment:
I think the reason we wanted people to use `deploy done` is because it would notify the next user. I'm just doing that directly here to make things roughly the same

Reply:
Nitpick: wouldn't `queue.isCurrent(user)` always be false here?

That sounds reasonable. I thought there was some logic like, "you would only take yourself out if you thought you weren't next to deploy".

That said, the interaction with the multiple queue is different from doing a simple deploy done, does it make sense to factor out the `deploy done` logic? It seems people conceptually think that `deploy done` and `remove me` are the same. But maybe people don't often use the multiple queue feature often anyway.

Reply:
> Nitpick: wouldn't queue.isCurrent(user) always be false here?

True, I'll change the conditional. 

> "you would only take yourself out if you thought you weren't next to deploy".

Right, I think that's how it was, but I disagree. `deploy done` makes it feel like you finished successfully. `remove me` is like, "I'm jumping out of the queue but I don't want it to seem like I succeeded"

I agree, the remove me logic I think is a bit special since I believe it removes all of your queued deploys if you have multiple. I think in practice though, people know what they're doing when they try to remove themselves. I was just trying to make it work in a reasonable way instead of telling them to use a different command